### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by Thermiee**
## Description
In this PR I implemented the views design:

### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### Milestone requirements, i:

- [x] Use preprocessed HTML file with embedded Ruby code.
- [x]Use layouts and templates for shared content.

Thank you for taking the time to review this pull request. :pray:

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,33 @@
-<h1>Here is the profile for a given a user</h1>
\ No newline at end of file
+<div class="center flex-column mt-3 mx-3">
+    <%= render partial: "shared/user", locals: {user: @user}%>
```
Comment (medaminedev66) [inline app/views/users/show.html.erb:2]: - [ ] When I click on the user link, it shows this error:

image

Please double-check that the correct path is rendered.

File: app/models/user.rb [hunk 2]
```diff
@@ -4,6 +4,6 @@ class User < ApplicationRecord
   has_many :likes
 
   def recent_posts
-    comments.order ('created_at DESC').limit(5)
+    posts.order ('created_at DESC').limit(5)
   end
```
Comment (medaminedev66) [inline app/models/user.rb]: - [ ] Another error shows up in the users/show.html.erb file:

image

Please remove the white space between "order" and "(" symbol.

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1 +1,33 @@
...
+    <div class="w-50 w-100-mobile mb-3 bg-light p-2 border">
+      <h3>Bio</h3>
+      <p><%= @user.bio %></p>
+    </div>
+    <div class="d-flex flex-column w-50 w-100-mobile">
+      <% @user.recent_posts.each do |post| %>
+      <div class="d-flex flex-column bg-light p-2 border mb-3">
+        <h5><%= post.title %></h5>
+        <p><%= post.text %></p>
```
Comment (medaminedev66) [inline app/views/users/show.html.erb:11]: - [ ] There is no column called `author_id` in the posts table. Please refactor your schema and ensure that the posts table has a column called `author_id`, not `user_id`.

image

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -1 +1,53 @@
...
+                  <% end %>
+                  <% end %>
+                </ul>
+              </div>
+            </div>
+          <% end %>
+        <% end %>
+      </div>
+        <%= button_to "Pagination", "", class: "btn btn-primary" %>
+    </div>
```
Comment (medaminedev66) [inline app/views/posts/index.html.erb]: - [ ] Please ensure that all the views are displayed without errors and ensure that the views are well associated with the Models and Controllers.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Thermiee  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- There are no linter errors ✔️
- You have a descriptive PR ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 4 above)
Comment (DeliceLydia) [reply app/views/posts/index.html.erb]: - [ ] I am still not able to create a post, in your post table you have `author_id` and `user_id` I am just confused about how you are creating a post, I think what you need is just a `user` who created a post, as I can see for your case foreign key should be `author_id`, not ` user_id` or you can clarify in your README how you are doing it

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Thermiee, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by afanimartin — APPROVED** [review]
Hi @Thermiee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**